src/Mod/Path/PathScripts/PathArray.py

```diff
@@ -52,6 +54,12 @@ def __init__(self, obj):
                         "Path", "The number of copies in Linear 1D and Polar pattern")
         obj.addProperty("App::PropertyVector", "Centre",
                         "Path", "The centre of rotation in Polar pattern")
+        obj.addProperty("App::PropertyBool", "SwapDirection",
```

Comment:
Please wrap all user-visible property descriptions in the QT_TRANSLATE_NOOP function so they get translated into other languages.   See 'pattern method' above.

Reply:
Will do